packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -215,6 +215,9 @@
         display: flex;
         justify-content: flex-start;
         font-weight: bold;
+        &--accumulators {
+            margin-left: -3px;
```

Comment:
@kate-deriv let's not adjust negative margin, instead please update grid values for `.dc-contract-card__grid {` class for Accumulators